src/test/unit/algorithms/SDRClassifierTest.cpp

```diff
@@ -117,19 +117,19 @@ TEST_F(SDRClassifierTest, SingleValue) {
   vector<UInt> input1{1u, 5u, 9u};
   vector<UInt> bucketIdxList{4u};
   vector<Real64> actValueList{34.7f};
-  ClassifierResult result1;
+  ClassifierResult result1; // TODO: REVIEW THIS TESTCASE, POTENTIAL SCOPING ISSUES?
   for (UInt i = 0u; i < 10u; ++i) {
-    ClassifierResult result1;
+    ClassifierResult result1; // SHADOWS?
```

Comment:
that's nasty! Good catch :eyeglasses:  
I'd say the inner result1 should be removed (?)

Reply:
Fixed.